zlp-scheduler/config/routes.rb

```diff
@@ -61,7 +61,7 @@
   
   get 'admin/view_result/:cohort_id', to: 'admin#view_result', as: 'view_result'
   get 'admin/view_conflicts/:cohort_id', to: 'admin#view_conflicts', as: 'view_conflicts'
-  
+  get 'admin/run_algorithm', to: 'admin#run_algorithm', as: 'run_algorithm'
```

Comment:
Is this a route you actually added, or was this deleted? I didn't see any changes to the admin controller.

Reply:
This is the change I tested for the new button, so it might be necessary here, I will delete it.